cert/spec/runner_spec.rb

```diff
@@ -73,5 +73,55 @@
       Cert.config = FastlaneCore::Configuration.create(Cert::Options.available_options, keychain_path: ".")
       Cert::Runner.new.revoke_expired_certs!
     end
+
+    describe ":filename option handling" do
+      filename = ""
+
+      let(:filename_ext) do
+        File.extname(filename) == ".cer" ? filename : "#{filename}.cer"
+      end
+
+      let(:generate) do
+        certificate = stub_certificate
+
+        allow(Spaceship).to receive(:login).and_return(nil)
+        allow(Spaceship).to receive(:client).and_return("client")
+        allow(Spaceship).to receive(:select_team).and_return(nil)
+        allow(Spaceship.client).to receive(:in_house?).and_return(false)
+        allow(Spaceship.certificate.production).to receive(:all).and_return([certificate])
+        allow(FastlaneCore::CertChecker).to receive(:installed?).and_return(true)
+
+        Cert.config = FastlaneCore::Configuration.create(Cert::Options.available_options, filename: filename)
+        Cert::Runner.new.launch
+      end
+
+      context 'with filename flag' do
+        it "can forget the file extension" do
+          filename = "cert_name"
+          expect(File.exist?(filename_ext)).to be false
+          generate
+          expect(File.exist?(filename_ext)).to be true
+          File.delete(filename_ext)
```

Comment:
One last question! Where do these files get created (and then deleted) from right now? Will these get generated **in** the specs folder? If so, can we tell cert to create them in a temp directory?

Reply:
Files are created where command is executed. 

Thus I can create a dedicated folder for this, but how should I manage it? Should I create and delete it each time a test is done? 

Moreover, maybe a global rule would be relevant for having all specs run into a tmp folder? this would be consistent and reusable.

How do you feel about it ?

Reply:
`cert` takes an option for `path` so I think pass a `Dir.tmpdir()` (or similar) into the `Cert.config`? 😊

Reply:
The reason I think this should be a temporary directory because right now, if the test fails, that file won't be deleted and will remain around